wagtail/admin/views/pages/delete.py

```diff
@@ -50,5 +77,10 @@ def delete(request, page_id):
             "page": page,
             "descendant_count": page.get_descendant_count(),
             "next": next_url,
+            "translation_count": translation_count,
+            "translation_descendant_count": sum(
+                [p.get_descendants().count() for p in pages_to_delete]
```

Comment:
We are only looking at translated pages as the message is "[..] and their combined N translated child pages". so `translation_descendant_count` needs to be `p.get_descendants().filter(alias_of__isnull=True).count()`